ui/lib/store.js

```diff
@@ -405,6 +405,15 @@ export const setKeys =
     return dispatch({ type: actionTypes.SET_KEYS, keys });
   };
 
+export const mesheryStore = {
+  organization: {
+    set: (organization) => (dispatch) =>
+      dispatch({ type: actionTypes.SET_ORGANIZATION, organization }),
```

Comment:
Directly use store.dispatch this won't work

Reply:
I guess we are not using this meshery store in the latest update. Sudhanshu we can remove it now.